views/splash.html

```diff
@@ -0,0 +1,95 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>{% if title %}{{ title }} :: {% endif %}Developers' Guild</title>
```

Comment:
This can be just `<title>{{ title }}</title>` or `<title>Welcome to Daze!</title>`